### PR TITLE
添加基于wechaty实现的基金小助手

### PR DESCRIPTION
**Pull request by xla145**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [ ] Add Blog Header, including title, author, date.  
    For example:  

    ``` md
    ---
    title: "添加基于wechaty实现的基金小助手"
    author: xula
    avatar : "https://avatars3.githubusercontent.com/u/18521352?s=400&u=0cd7fac67d8159796fa99921939dabaea3f00643&v=4"
   date: 2020-09-05
    ---
    add blog
    ```
Here is the add files:
- 2020-09-05-fund-wechay-python-bot.md: https://github.com/wechaty/wechaty.github.io/blob/master/_post/2020-09-05-fund-wechay-python-bot.md
- picDir: https://github.com/wechaty/wechaty.github.io/blob/master/assets/2020/fund-wechay-python

update files:

- authors.yml: https://github.com/wechaty/wechaty.github.io/blob/master/_data/authors.yml

  For excample:
  ```md
  ---
  name: Jiarui LI (李佳芮)
  site: https://lijiarui.github.io
  bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
  avatar: /assets/developers/lijiarui/avatar.png
  email: rui@juzi.bot
  twitter: https://weibo.com/u/2175505900
  ---
  ```
  Here is the example file:
  - lijiarui.md: https://github.com/wechaty/wechaty.github.io/blob/master/_developers/lijiarui.md
  - It will shown like this on wechaty blog: https://wechaty.github.io/developers/lijiarui/

To read more about the contribute guideline, visit: <https://github.com/wechaty/wechaty.js.org#contribute-guideline>

### CODE REVIEW

**Comment by xla145** [comment]
Please review

**Comment by lijiarui** [comment]
Please sign the CLA first, and update your author data file at <https://github.com/wechaty/wechaty.js.org/blob/master/_developers/YOUR_GITHUB_USERNAME.md> with your GitHub username.

**Comment by xla145** [comment]
Ok,thanks

**Comment by xla145** [comment]
I've added the developer information, please review

File: _data/authors.yml [hunk 1]
```diff
@@ -923,3 +923,18 @@ csjuXYZ:
     - label: "Email"
       icon: "fas fa-fw fa-envelope-square"
       url: "hoc0415@foxmail.com"
+
+xula145:
+  name        : "徐利安"
+  bio         : "基于wechaty实现的基金小助手"
+  avatar      : "https://avatars3.githubusercontent.com/u/18521352?s=400&u=0cd7fac67d8159796fa99921939dabaea3f00643&v=4"
+  links:
```
Comment (lijiarui) [inline _data/authors.yml]: Please remove this file since we change to developers markdown file.

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request.

**Comment by xla145** [comment]
I have fininshed removing authors file

**Comment by lijiarui** [comment]
Please sign the CLA

**Comment by lijiarui** [comment]
@xla145 Please sign the CLA first and then I can review your PR.

**Comment by wj-Mcat** [comment]
ping @xla145  Please sign the CLA first and then I can review your PR. If you have not do it at next three days, we will close this pr. Thanks a lot.

**Comment by huan** [comment]
Please sign the CLA by clicking the `[CLA | not signed yet]` button above, so that we can continue processing your PR.

**Comment by huan** [comment]
Close this PR due to inactivity after 30 days.

**Pull request closed without merging**